data/model/notification.py

```diff
@@ -263,6 +265,11 @@ def list_repo_notifications(namespace_name, repository_name, event_name=None):
             .where(ExternalNotificationEvent.name == event_name)
         )
 
+    if notification_uuid:
+        query = query.switch(RepositoryNotification).where(
```

Comment:
i don't think you need this switch since `RepositoryNotification` is the table being selected from

Reply:
yes, fixed in the latest commit.